analytics_data_api/v0/models.py

```diff
@@ -206,3 +208,22 @@ class Video(BaseVideo):
 
     class Meta(BaseVideo.Meta):
         db_table = 'video'
+
+
+if settings.ELASTICSEARCH_LEARNERS_HOST:
+    connections.connections.create_connection(hosts=[settings.ELASTICSEARCH_LEARNERS_HOST])
+
+# By creating the roster index, we can use the @roster.doc_type decorator below
+roster = Index(settings.ELASTICSEARCH_LEARNERS_INDEX)
+
+
+@roster.doc_type
+class RosterEntry(DocType):
```

Comment:
I'm a little surprised you don't have to declare the fields here. Are there pros and cons associated with declaring them or not?

Part of me likes the explicitness of declaring them.